### PR TITLE
Some pep8 cleanup of sympy.physics.vector.

### PR DESCRIPTION
**Pull request by moorepants**
#### Brief description of what is fixed or changed

Simple pep8 code cleanup, fixing linting warnings.

#### Release Notes

NO ENTRY

### CODE REVIEW

File: sympy/physics/vector/frame.py [hunk 1]
```diff
@@ -130,7 +130,8 @@ def __init__(self, name, indices=None, latexs=None, variables=None):
         >>> vlatex(P.x)
         'A1'
 
-        symbols() can be used to create multiple Reference Frames in one step, for example:
+        ``symbols()`` can be used to create multiple Reference Frames in one
```
Comment (oscargus) [inline sympy/physics/vector/frame.py:133]: Even better is probably `:func:~.symbols` (with correct backticks around `~.symbols`, but not sure how to get those here...)

File: sympy/physics/vector/point.py [hunk 2]
```diff
@@ -32,7 +32,8 @@ class Point:
     >>> O.acc(N)
     u1'*N.x + u2'*N.y + u3'*N.z
 
-    symbols() can be used to create multiple Points in a single step, for example:
+    ``symbols()`` can be used to create multiple Points in a single step, for
```
Comment (oscargus) [inline sympy/physics/vector/point.py:35]: Same here. Possible also for Point, but maybe that looks a bit weird as the full path must be included (I think?).

**Comment by oscargus** [comment]
Looks good! Some minor comment, but nothing really required.

File: sympy/physics/vector/frame.py (hunk 1 above)
Comment (moorepants) [reply sympy/physics/vector/frame.py:133]: I'll work on those in other PRs.

**Comment by moorepants** [comment]
Thanks. I'll merge and work on the cross referencing later.

**Pull request merged**